src/explore-education-statistics-admin/src/pages/publication/PublicationContactPage.tsx

```diff
@@ -88,7 +62,7 @@ const PublicationContactPage = () => {
               {contact.contactTelNo}
             </SummaryListItem>
           </SummaryList>
-          {publication.permissions.canUpdatePublication && (
+          {contact.permissions?.canUpdatePublication && (
```

Comment:
As mentioned in another comment, the permission for updating a contact should be on the `publication` itself i.e. we should have something like `publication.permissions.canUpdateContact`.

Reply:
I've also changed `getContact` and added a `ContactWithPermissions` interface so I can remove the `?` here.

Reply:
Turns out I cannot do that because `UpdateContact` just returns a `Contact` and we pass the result of `UpdateContact` to `setContact` that was expecting a `ContactWithPermissions`. I've left the extra code around `getContact` and the new `ContactWithPermissions` interface.

Reply:
Would really just move the contact permission onto the publication as part of this PR to save yourself the trouble of having to do it later!